cpp/dependency_scan/metadata/README.md

```
@@ -0,0 +1,758 @@
+git diff metadata/
+python ../scan_build_artifacts.py --validate
+```
+
+## Advanced Usage
+
+### Wildcards and Regex
+
+Patterns are exact or substring matches - **no wildcards or regex supported**.
+
+Instead of:
+```yaml
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)